src/branding/centos.scss

```diff
@@ -0,0 +1,23 @@
+@use "standard.scss" as *;
+
+:root {
+  /* Defaults to a "Fedora remix"; if not a remix, set "false" */
+  --remix: false;
+
+  /*
+  Brand colors
+  */
+  --brand-primary: black;
+  --brand-text: white;
+
+  /*
+  The actual logo SVG.
+
+  First, optimize the SVG with SVGO; there's a web-based version
+  called SVGOMG at https://jakearchibald.github.io/svgomg/
+
+  Then, encode using URI encoding (not base64) and use as a
+  background. There's a useful tool at https://svgencode.com/
+  */
+  --logo: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 128 128'%3E%3Cg stroke-width='.3' paint-order='fill markers stroke' transform='matrix(3.1840795,0,0,3.1840795,-1171.916,804.24614)'%3E%3Ccircle cx='435.1' cy='-124.1' r='20.1' fill='%23a14f8c' transform='rotate(-15)'/%3E%3Cpath fill='%23fff' d='m388-248-4 5h-7v6l-4 4 4 5v6h7l4 5 5-5h6v-6l4-5-4-4-6-6zm1 5v5l-1 1-1-1v-5zm-8 2 4 3v2h-2l-3-3zm-4 7h5l2 1-2 2h-5zm17 0h5v3h-5l-1-2zm-11 5h2v2l-3 3-2-2zm8 0h2l4 3-2 2-4-3zm-3 1 1 1v5h-2v-5z'/%3E%3Cpath fill='%23efa724' d='m389-236-1 2-1-2v2h-2l1 1-1 2h2v2l1-2 1 2v-2h2l-1-2 1-1h-2zm4-7 2 2-4 3v2h2l4-3 2 2v-6z'/%3E%3C/g%3E%3C/svg%3E");
```

Comment:
@garrett I don't think we can include logos like this into out packaging.

See https://github.com/cockpit-project/cockpit/blob/main/doc/branding.md#branding